CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - oneview_storage_system
 - oneview_uplink_set
 - oneview_volume
+- oneview_volume_attachment
```

Comment:
This should come under New Resources Support heading.

Reply:
done